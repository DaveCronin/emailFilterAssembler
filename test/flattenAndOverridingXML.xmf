a1 = {to "1"}
a2 = {to "2"}
a3 = {to "3"}
a4 = {to "4"}
a5 = {to "5"}
a6 = {to "6"}

pattern1 = [star, archive]
pattern2 = [mark as read]
pattern3 = [archive, label "veryimportant"]
pattern4 = [not spam, label "reallyimportant"]
pattern5 = [delete]
pattern6 = [important]

if a1 -> pattern1
	if a2 -> pattern2
	if a3 -> pattern3
		if a4 -> pattern4
		if a5 -> pattern5
			if a6 -> pattern6