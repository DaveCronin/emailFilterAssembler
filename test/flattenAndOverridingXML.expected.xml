<?xml version='1.0' encoding='UTF-8'?><feed xmlns='http://www.w3.org/2005/Atom' xmlns:apps='http://schemas.google.com/apps/2006'>
	<title>Mail Filters</title>
	<id>tag:mail.google.com,2008:filters:4366218717592519022</id>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='1 &amp; 2'/>
		<apps:property name='shouldMarkAsRead' value='true'/>
		<apps:property name='shouldStar' value='true'/>
		<apps:property name='shouldArchive' value='true'/>
	</entry>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='1 &amp; 3 &amp; 4'/>
		<apps:property name='label' value='reallyimportant'/>
		<apps:property name='shouldStar' value='true'/>
		<apps:property name='shouldArchive' value='true'/>
		<apps:property name='shouldNeverSpam' value='true'/>
	</entry>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='1 &amp; 3 &amp; 5 &amp; 6'/>
		<apps:property name='label' value='veryimportant'/>
		<apps:property name='shouldStar' value='true'/>
		<apps:property name='shouldTrash' value='true'/>
		<apps:property name='shouldArchive' value='true'/>
		<apps:property name='shouldAlwaysMarkAsImportant' value='true'/>
	</entry>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='1 &amp; 3 &amp; 5'/>
		<apps:property name='label' value='veryimportant'/>
		<apps:property name='shouldStar' value='true'/>
		<apps:property name='shouldTrash' value='true'/>
		<apps:property name='shouldArchive' value='true'/>
	</entry>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='1 &amp; 3'/>
		<apps:property name='label' value='veryimportant'/>
		<apps:property name='shouldStar' value='true'/>
		<apps:property name='shouldArchive' value='true'/>
	</entry>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='1'/>
		<apps:property name='shouldStar' value='true'/>
		<apps:property name='shouldArchive' value='true'/>
	</entry>
</feed>